application-admintools-api/src/main/java/com/xwiki/admintools/configuration/AdminToolsConfiguration.java

```diff
@@ -22,16 +22,13 @@
 import java.util.List;
 
 import org.xwiki.component.annotation.Role;
-import org.xwiki.stability.Unstable;
 
 /**
  * Admin Tools configurations.
  *
  * @version $Id$
- * @since 1.0
  */
 @Role
-@Unstable
```

Comment:
This should not be removed, please check if it's missing somewhere else too https://dev.xwiki.org/xwiki/bin/view/Community/DevelopmentPractices#H40UnstableAnnotation